src/views/Player.vue

```diff
@@ -229,7 +229,7 @@ export default {
         )
         .then((res) => {
           console.log(res.data);
-          var questions = res.data.plioDetails.questions.questions;
+          var questions = res.data.plioDetails.items;
```

Comment:
Shouldn't you check for the type of items here? Questions should be only those that have type `question` no?

Reply:
I haven't done that currently intentionally because we don't know how other items could look like and correspondingly, how should answers look then. Since I was a bit unsure of how any new items could look like, I chose to keep this as is and make that specific change when we do end up adding another item type. What do you think?

Reply:
Questions should still be only those that have type question, IMO. Rest of items will probably end up having their own component, I think!

Reply:
Yep. And I think this variable itself will become `items` in the future and we'll either use PlioQuestion or another component as necessary!

Reply:
I know this is merged, but you should rename the variable also to `items` and then just set `questions` to items. The important thing here is that the frontend follows the structure of the JSON as closely as possible. This is what will prevent bugs in the future.

Reply:
Makes sense. Will do 👍